catalog/core/catalog-core-ftp/src/main/java/ddf/catalog/core/ftp/FtpServerStarter.java

```diff
@@ -36,8 +40,16 @@
 
     private static final String DEFAULT_LISTENER = "default";
 
+    private static final String WANT_AUTH = "WANT";
```

Comment:
`WANT` doesn't require 2-way SSL, if we want to enforce 2-way this should be `NEED` and you should use the enum that already exists in the mina project.